src/components/Community/data.json

```diff
@@ -44,26 +44,26 @@
     }
   ],
   "userContent": [
+    {
+      "url": "https://medium.com/acing-ai/ml-ops-data-science-version-control-5935c49d1b76",
+      "title": "ML Ops: Data Science Version Control",
+      "author": "Vimarsh Karbhari",
+      "date": "2020-04-23",
+      "pictureUrl": "/img/community/ugc/medium_logo.png"
+    },
+    {
+      "url": "https://blog.codecentric.de/en/2020/01/remote-training-gitlab-ci-dvc/",
+      "title": "Manage your Data Science Project in R",
+      "author": "Marcel Ribeiro-Dantas",
+      "date": "2020-03-05",
+      "pictureUrl": "/uploads/avatars/marcel.jpg"
```

Comment:
one more small ask, @andronovhopf  :) please let's duplicate this image to the `/img/community/ugc` or something inside `/img/community` - it will be easier to manage these assets - if we don't mix them.

Reply:
great! doing it now

Reply:
done